api/_lib/template.ts

```diff
@@ -77,6 +80,10 @@ function getCss(theme: string, fontSize: string) {
         margin: 0 75px;
     }
 
+    .logo-rounded {
+        border-radius: 9999px;
```

Comment:
```suggestion
        border-radius: 50%;
```

I think this is a common pattern

Reply:
@styfle yeah I think both is fine in this case. It has different results for non-square images, but I don't think that matters here.
https://github.com/tailwindlabs/tailwindcss/issues/1949